src/renderer/components/playlist-info/playlist-info.js

```diff
@@ -412,6 +415,41 @@ export default defineComponent({
       showToast(this.playlistDeletionDisabledLabel)
     },
 
+    handlePlaylistExport: async function () {
+      const dateStr = getTodayDateStrLocalTimezone()
+      const title = this.selectedUserPlaylist.playlistName.replaceAll(' ', '-')
```

Comment:
IMO spaces are usually replaced with `_` more often then `-`
I think only for SEO (URL) people prefer using `-`
Let me know if that's wrong :)

Also we might need to replace all invalid chars, e.g. `/` on MacOS, different for other OSes
(handling user input is nasty)

![image](https://github.com/user-attachments/assets/d955f900-f215-4143-b3e1-d02d52ea2c02)

```suggestion
      const title = this.selectedUserPlaylist.playlistName.replaceAll(' ', '_')
```